src/updateHttpApiStageThrottling.js

```diff
@@ -50,7 +50,7 @@ const createRouteSettingsForHttpApiEndpoint = (endpointSettings, serverless) =>
 
     let routeSettings = {};
```

Comment:
The code can be simplified so that the method returns just `routeSettingsForMethod(path, method, endpointSettings)` from this point on.

Reply:
Please kindly check my changes. I have left the conversion to upper case unhanged. 
Thanks.